pkg/cli/admin/nodeimage/nodeimage.go

```diff
@@ -0,0 +1,23 @@
+package nodeimage
+
+import (
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+func NewCmdNodeImage(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "node-image",
+		Short: "Add nodes to an existing cluster.",
+		Long: templates.LongDesc(`
```

Comment:
I'd recommend to define these in consts like https://github.com/openshift/oc/blob/a2e5b09d7d14b77c14f1b32550f8c5a0328e73b3/pkg/cli/admin/mustgather/mustgather.go#L56-L82 otherwise, it is difficult to manage raw strings in here.

Reply:
Sounds good. Thank you.